docs/migrations.rst

```diff
@@ -125,13 +123,6 @@ Please be aware that when a ``change`` method exists, Phinx will automatically
 ignore the ``up`` and ``down`` methods. If you need to use these methods it is
 recommended to create a separate migration file.
 
-.. note::
```

Comment:
Why has this been removed?

Reply:
Because it is not true anymore after my previous PR

Reply:
Ha! I'll shut up then!